### PR TITLE
[TFA] Updating testcase timeout for sanity_rgw and sanity_rgw_ms

### PR DESCRIPTION
**Pull request by anrao19**
# Description
Default timeout is 3600, for testcase which takes more than 3600s timeout has been added, for othere timeout is 3600s

testcasess like 
1. bucket Granular related
2.  test mdlog trimming on primary
3. disable and enable dynamic resharding for 10K buckets
4.  test_sync_on_bucket_with_0_shards_0 timeout https://issues.redhat.com/browse/RHCEPHQE-15378 

pass log: http://magna002.ceph.redhat.com/cephci-jenkins/Anuchaithra/cephci-run-1AOUWG/

All test logs located here: /tmp/cephci-run-CQOAV3

All test logs located here: /tmp/cephci-run-CQOAV3

TEST NAME                                 TEST DESCRIPTION                                                      DURATION                                      STATUS                                 COMMENTS
setup pre-requisites                       Install software pre-requisites for cluster deployment.                   0:11:05.749536                   
    Pass
deploy cluster                   RHCS cluster deployment using cephadm.                         0:25:45.582169                   Pass
configure client                 Configure the RGW client system                                0:00:55.847861                   Pass
setup multisite                  Setting up RGW multisite replication environment               0:03:11.252015                   Pass
get shared realm info on prima   Retrieve the configured environment details                    0:00:10.822430                   Pass
get shared realm info on secon   Retrieve the configured environment details                    0:00:12.137808                   Pass
create tenanted user             create tenanted user                                           0:30:21.169629                   Pass
Bucket generation verification   Bucket generation verification                                 0:25:40.231664                   Pass
Disable DBR feature on zonegro   Disable DBR feature on zonegroup                               0:26:45.288849                   Pass
Re-enable DBR feature on clust   Re-enable DBR feature on cluster an verify                     0:21:40.570059                   Pass
Dynamic Resharding tests on Pr   Test dynamic resharding on greenfield deployment               0:07:54.794191                   Pass
Dynamic Resharding tests on Pr   Test dynamic resharding on greenfield deployment tenanted us   0:05:47.962245                   Pass
Dynamic Resharding tests on Pr   Test dynamic resharding on versioned bucket                    0:08:37.606394                   Pass
Dynamic Resharding tests on ve   Test dynamic resharding on versioned bucket and tenanted use   0:08:38.283779                   Pass
Dynamic Resharding on versione   Resharding test - dynamic resharding on versioned bucket       0:20:50.177707                   Pass
test exceeding quota limit on    test exceeding quota limit on dynamically resharded bucket a   0:05:18.079560                   Pass
Manual Resharding tests on Pri   Test sync enable and disable with manual resharding            0:25:42.965222                   Pass
Perform zone deletion in maste   Test zone deletion in master                                   0:00:10.742936                   Pass
bucket granular sync policy wi   Test bucket granular sync with sync from differnt bucket       1:14:51.972629                   Pass
Perform realm rename in master   Test realm rename in master                                    0:00:12.786307                   Pass
bucket granular sync policy wi   Test bucket granular sync with sync to differnt bucket         1:14:58.860217                   Pass

2024-07-25 16:16:25,415 (cephci.sanity_rgw_multisite)

https://github.com/anrao19/cephci/blob/master/tests/rgw/sanity_rgw_multisite.py#L199
https://github.com/anrao19/cephci/blob/master/tests/rgw/sanity_rgw.py#L184 

bucket granular passed, but due to timeout
ceph.ceph.SocketTimeoutException: sudo venv/bin/python /home/cephuser/rgw-ms-tests/ceph-qe-scripts/rgw/v2/tests/s3_swift/test_multisite_bucket_granular_sync_policy.py -c rgw-ms-tests/ceph-qe-scripts/rgw/v2/tests/s3_swift/multisite_configs/test_multisite_granular_bucketsync_forbidden_allowed.yaml --rgw-node 10.0.195.44 failed to complete within 3600s

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: suites/quincy/rgw/tier-1_rgw_multisite.yaml [hunk 1]
```diff
@@ -295,21 +294,20 @@ tests:
...
+      desc: Test bucket granular sync policy with allowed allowed semantic
+      polarion-id: CEPH-83575868
+      module: sanity_rgw_multisite.py
+      comments: Known issue BZ-2153834 targeted to 8.0
+      clusters:
+        ceph-pri:
+          config:
+            script-name: test_multisite_bucket_granular_sync_policy.py
+            config-file-name: test_multisite_granular_bucket_sync_policy.yaml
+            verify-io-on-site: ["ceph-sec"]
```
Comment (psathyan) [inline suites/quincy/rgw/tier-1_rgw_multisite.yaml:309]: Please add this as part of separate commit in this PR. Adding details of enablement.

Comment (anrao19) [reply suites/quincy/rgw/tier-1_rgw_multisite.yaml:309]: @psathyan , This was a working testcase. commented due to BZ.
So included this as a part of this pr

**Review by psathyan — COMMENTED** [review]
/lgtm

File: suites/reef/rgw/tier-1_rgw_upgrade_7_0GA_to_7x_latest.yaml [hunk 2]
```diff
@@ -84,8 +84,7 @@ tests:
       module: sanity_rgw.py
       config:
         script-name: ../aws/test_sts_rename_large_object.py
-        config-file-name: ../../aws/configs/test_sts_rename_large_object.yaml
-        timeout: 500
+        config-file-name: ../../aws/configs/test_sts_rename_large_object.yaml\
```
Comment (ckulal) [inline suites/reef/rgw/tier-1_rgw_upgrade_7_0GA_to_7x_latest.yaml:87]: please remove additional \ at the end

Comment (anrao19) [reply suites/reef/rgw/tier-1_rgw_upgrade_7_0GA_to_7x_latest.yaml]: done

**Review by mkasturi18 — APPROVED** [review]

**Pull request merged**